### PR TITLE
Implement Jest tests on coworkings, teams and coworkers  files

### PR DESCRIPTION
**Pull request by juliekohl**

### CODE REVIEW

File: src/api/coworkers.spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    it("should post update a coworkers by its name or pass", async () => {
+        const response = await request(app).post("/coworkers/2").send({
+            name: "Name",
+            password: "secret"
+        });
+        expect(response.body.message).toBe("Success");
+    })
+
+    it("should delete a coworkers by its id", async () => {
```
Comment (cesarkohl) [inline src/api/coworkers.spec.ts:40]: Before deleting, let's add a coworker here on the test to make sure the result will always be success.

File: src/api/coworkings.spec.ts [hunk 2]
```diff
@@ -16,10 +16,22 @@ describe("coworkings", () => {
         expect(response.body.id).toBe(1);
     })
 
-    it("should post create a coworking by its id and name", async () => {
+    it("should post create a coworking by its name", async () => {
```
Comment (cesarkohl) [inline src/api/coworkings.spec.ts:19]: Replace `should post create` with `should create`.

File: src/api/coworkings.spec.ts [hunk 3]
```diff
@@ -16,10 +16,22 @@ describe("coworkings", () => {
...
         const response = await request(app).post("/coworking").send({
             name: "Name"
         });
-        expect(typeof response.body).toBe("number");
+        expect(response.body).toBe("number");
```
Comment (cesarkohl) [inline src/api/coworkings.spec.ts:23]: `typeof response.body`

File: src/api/coworkings.spec.ts [hunk 4]
```diff
@@ -16,10 +16,22 @@ describe("coworkings", () => {
...
+    })
+
+    it("should post update a coworking by its name", async () => {
+        const response = await request(app).post("/coworking/1").send({
+            name: "Name"
+        });
+        expect(response.body).toBe("number");
```
Comment (cesarkohl) [inline src/api/coworkings.spec.ts:30]: typeof response.body

File: src/api/coworkings.spec.ts [hunk 5]
```diff
@@ -16,10 +16,22 @@ describe("coworkings", () => {
...
-    it("should post create a coworking by its id and name", async () => {
+    it("should post create a coworking by its name", async () => {
         const response = await request(app).post("/coworking").send({
             name: "Name"
         });
-        expect(typeof response.body).toBe("number");
+        expect(response.body).toBe("number");
+    })
+
+    it("should post update a coworking by its name", async () => {
```
Comment (cesarkohl) [inline src/api/coworkings.spec.ts:26]: Replace `should post update` with `should update`.

File: src/api/coworkings.spec.ts [hunk 6]
```diff
@@ -16,10 +16,22 @@ describe("coworkings", () => {
...
+        const response = await request(app).post("/coworking/1").send({
+            name: "Name"
+        });
+        expect(response.body).toBe("number");
+    })
+
+    it("should delete a coworking by its id", async () => {
```
Comment (cesarkohl) [inline src/api/coworkings.spec.ts:33]: First of all, create a new coworking. Then, delete it.

File: src/api/teams.spec.ts [hunk 7]
```diff
@@ -0,0 +1,44 @@
+import request from 'supertest';
+import app from 'app';
+
+describe("teams", () => {
+    it('sanity test', () => {
```
Comment (cesarkohl) [inline src/api/teams.spec.ts:5]: Remove sanity test.

File: src/api/teams.spec.ts [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+        expect(true).toBe(true);
+    })
+
+    it("should respond with a 200 status code", async () => {
```
Comment (cesarkohl) [inline src/api/teams.spec.ts:9]: Remove this test.

File: src/api/teams.spec.ts [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+        const response = await request(app).get("/teams");
+        expect(response.statusCode).toBe(200);
+    })
+
+    it("should get a teams by its email and name", async () => {
```
Comment (cesarkohl) [inline src/api/teams.spec.ts:14]: Team in singular, not plural: "should get a team".

File: src/api/teams.spec.ts [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+        const response = await request(app).get("/teams/1");
+        expect(response.body).toStrictEqual({
```
Comment (cesarkohl) [inline src/api/teams.spec.ts:16]: Expect not result empty.

File: src/api/teams.spec.ts [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+            email: "Amos38@gmail.com",
+            name: "Sharon Mills"
+        });
+    })
+
+    it("should post create a teams by its name, email and pass", async () => {
```
Comment (cesarkohl) [inline src/api/teams.spec.ts:22]: Replace `should post create` with `should create`.
Replace `pass` with `password`.

File: src/api/teams.spec.ts [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+        const response = await request(app).post("/teams").send({
+            coworking_id: 6,
+            name: "Name8",
+            email: "email-test8@email.com",
+            password: "secret"
+        });
+        expect(response.body.message).toBe("Success");
+    })
+
+    it("should post update a teams by its name or pass", async () => {
```
Comment (cesarkohl) [inline src/api/teams.spec.ts:32]: Replace `should post update` with `should update`.
Replace `pass` with `password`.

File: src/api/teams.spec.ts [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+        const response = await request(app).post("/teams/2").send({
+            name: "Name",
+            password: "secret"
+        });
+        expect(response.body.message).toBe("Success");
+    })
+
+    it("should delete a teams by its id", async () => {
```
Comment (cesarkohl) [inline src/api/teams.spec.ts:40]: First of all, create a team. Then, delete it.

**Review by cesarkohl — CHANGES_REQUESTED** [review]

File: src/api/coworkers.spec.ts [hunk 14]
```diff
@@ -0,0 +1,44 @@
+import request from 'supertest';
+import app from 'app';
+
+describe("coworkers", () => {
+    it('sanity test', () => {
```
[the remark this answers is not in the document]
Comment (juliekohl) [reply src/api/coworkers.spec.ts:5]: done.

File: src/api/coworkers.spec.ts [hunk 15]
```diff
@@ -0,0 +1,44 @@
...
+
+    it("should get a coworkers by its email and name", async () => {
+        const response = await request(app).get("/coworkers/1");
+        expect(response.body).toStrictEqual({
+            email: "Sedrick.Johnston@gmail.com",
+            name: "Kristine Moore"
+        });
+    })
+
+    it("should post create a coworkers by its name, email and pass", async () => {
```
[the remark this answers is not in the document]
Comment (juliekohl) [reply src/api/coworkers.spec.ts:22]: done.

File: src/api/coworkers.spec.ts [hunk 16]
```diff
@@ -0,0 +1,44 @@
...
+        const response = await request(app).post("/coworkers").send({
+            coworking_id: 6,
+            name: "Name8",
+            email: "email-test8@email.com",
+            password: "secret"
+        });
+        expect(response.body.message).toBe("Success");
+    })
+
+    it("should post update a coworkers by its name or pass", async () => {
```
[the remark this answers is not in the document]
Comment (juliekohl) [reply src/api/coworkers.spec.ts:32]: done.

File: src/api/coworkers.spec.ts [hunk 17]
```diff
@@ -0,0 +1,44 @@
...
+        const response = await request(app).post("/coworkers/2").send({
+            name: "Name",
+            password: "secret"
+        });
+        expect(response.body.message).toBe("Success");
+    })
+
+    it("should delete a coworkers by its id", async () => {
+        const response = await request(app).delete("/coworkers/6")
+        expect(response.headers['content-type']).toEqual(expect.stringContaining("json"))
```
[the remark this answers is not in the document]
Comment (juliekohl) [reply src/api/coworkers.spec.ts:42]: done.

File: src/api/teams.spec.ts (hunk 9 above)
Comment (juliekohl) [reply src/api/teams.spec.ts:14]: done.

File: src/api/teams.spec.ts (hunk 11 above)
Comment (juliekohl) [reply src/api/teams.spec.ts:22]: done.

File: src/api/teams.spec.ts (hunk 12 above)
Comment (juliekohl) [reply src/api/teams.spec.ts:32]: done.

File: src/api/teams.spec.ts (hunk 13 above)
Comment (juliekohl) [reply src/api/teams.spec.ts:40]: done.

File: src/api/teams.spec.ts (hunk 10 above)
Comment (juliekohl) [reply src/api/teams.spec.ts]: Done.

File: src/api/teams.spec.ts (hunk 11 above)
Comment (cesarkohl) [reply src/api/teams.spec.ts]: Replace should post create with should create.

**Review by cesarkohl — APPROVED** [review]

**Pull request merged**